subiquity/server/server.py

```diff
@@ -200,6 +200,7 @@ def get_installer_password_from_cloudinit_log():
     })
 
 POSTINSTALL_MODEL_NAMES = ModelNames({
+    "drivers",
```

Comment:
We could choose to make the drivers controller one that does not have to be configured, which would simplify some of the force-configuration in the controller, but that depends on if the Server TUI will have the drivers UI element or not.

Reply:
Thanks! As mentioned in another comment, we're going to have a UI component for server (skipped if no drivers are found) so I don't think we can go for the that construct. 
Having said that, I'll keep the option in mind. I think it can be useful for future components to come :)